pages/api/hunks.ts

```diff
@@ -0,0 +1,16 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveHunk } from '../../utils/db/relevance';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const hunkjson = req.body;
+  console.info("Saving hunks...", hunkjson);
```

Comment:
Wouldn't this be a big object? (Thinking about the importance of the info log, vs. the performance and storage load).